src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/stats/MflStatSpec.groovy

```diff
@@ -353,4 +370,35 @@ class MflStatSpec extends HealthCheckSpecification {
             flowHelperV2.deleteFlow(flow.flowId)
         }
     }
+
+    def switchIsConnectedToFl(SwitchId switchId, Boolean mgmt, Boolean stats) {
+        Wrappers.wait(WAIT_OFFSET / 2) {
+            def switchConnectionInfo = northboundV2.getSwitchConnections(switchId)
+            if (mgmt && stats) {
+                assert !findMgmtFl(switchConnectionInfo).empty
+                assert !findStatFl(switchConnectionInfo).empty
+            } else if (mgmt && !stats) {
+                assert !findMgmtFl(switchConnectionInfo).empty
+                assert findStatFl(switchConnectionInfo).empty
+            } else if (!mgmt && stats) {
+                assert findMgmtFl(switchConnectionInfo).empty
+                assert !findStatFl(switchConnectionInfo).empty
+            } else if (!mgmt && !stats) {
+                assert findMgmtFl(switchConnectionInfo).empty
+                assert findStatFl(switchConnectionInfo).empty
+            }
```

Comment:
```
def switchConnectionInfo = northboundV2.getSwitchConnections(switchId)
assert !findMgmtFl(switchConnectionInfo).empty == mgmt
assert !findStatFl(switchConnectionInfo).empty == stats
```
and remove all ifs

Reply:
It is so easy, fantastic. You made my day.
Refactored.